recipes/qt/5.x.x/test_package/conanfile.py

```diff
@@ -43,17 +43,17 @@ def _getenvpath(var):
                         os.environ[var] = val
                     return val
 
-                value = _getenvpath('CC')
+                value = _getenvpath("CC")
                 if value:
-                    args += ['QMAKE_CC=' + value,
-                             'QMAKE_LINK_C=' + value,
-                             'QMAKE_LINK_C_SHLIB=' + value]
+                    args += ['QMAKE_CC="' + value + '"',
```

Comment:
Yuck, this probably break if someone has compiler flags set on linux

Reply:
okay, it's test package, so it's not that critical I guess

Reply:
At least it's coherent with the recipe

Reply:
> Yuck, this probably break if someone has compiler flags set on linux... though I am certain in helps windows
> 
> I am not sure which way to go 🤔 we'll see what other say

As @SSE4 and @ericLemanissier mentioned, this change only makes the syntax of "test_package.conanfile.py" identical to the "conanfile.py" of the package for Qt5.

What are your specific concerns?

PS.: I'm using Linux as a host for cross-compiling with a Yocto SDK.

Reply:
@prince-chrismc : Marked this conversation as resolved. If you still have concerns, please let me know.

Reply:
Nop, sounds good to me!

Glad this was resolved quickly, thanks everyone for your input ❤️